types/src/on_chain_config/aptos_features.rs

```diff
@@ -60,10 +60,20 @@ pub struct Features {
 
 impl Default for Features {
     fn default() -> Self {
```

Comment:
this should probably be removed , in favor of default_features(), @runtian-zhou  (for the separate PR)

Reply:
I don't mind just merging it in. As long as we make it a separate commit message?

Reply:
I mean it is more convoluted than just removing this default - as behavior of move unit tests will change, etc. 

So it's better for that to be done separately, and consequences adjusted

Reply:
Let's do it separately then? Why do we need `default_features` over default impl?

Reply:
Makes sense. We just need a centralized location for all the default features that will be enabled in the genesis